Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -185,7 +183,6 @@ void EbrApplicationActivated(IActivatedEventArgs^ args) {
 
 void _ApplicationMainLaunch(ActivationType activationType, Platform::Object^ activationArg) {
     _ApplicationLaunch(activationType, activationArg);
-
     _appEvents = ref new AppEventListener();
     _appEvents->_RegisterEventHandlers();
```

Comment:
Are you sure? We can call AppEventListener() after we do ref new App() with the old activation

---
In reply to: [72352741](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72352741) [](ancestors = 72352741,72338254)